values-schema.yaml

```diff
@@ -822,8 +821,19 @@ definitions:
       id:
         readOnly: true
         type: string
+      clusterWide:
+        description: Will create the secret in all namespaces.
+        type: boolean
+        default: false
+      teamWide:
```

Comment:
It would be better to call it `allTeams` as teamWide suggests it is for one team.